### ISSUE TITLE
Add Inbox for viewing all unreads

### ISSUE THREAD

**Issue by ndarilek** [vector-im/element-meta#1375]
**Is your suggestion related to a problem? Please describe.**
I currently participate in a number of Discord servers. I find it incredibly useful to skim all the new messages in channels that I'm in periodically as a way to keep tabs on the communities I'm in. If something happens in a Discord community, I am likely to know about it in short order. But if something happens in a Matrix room, I don't know about it unless I explicitly open the room, find the earliest unread message, and start reading. It's doable, but definitely not the same river of messages that makes Discord communities much easier to follow.

**Describe the solution you'd like**
Discord has an [inbox](https://support.discord.com/hc/en-us/articles/360045027712-Inbox-FAQ) which displays all unread messages sorted by server and channel. This is noisy and requires some curation to get right, but Matrix has an edge here in that joining an individual community doesn't imply joining every channel in that community to which you have access. Being able to read all my messages in a single stream, then mark them as read when I'm done, would help Matrix to feel like the same community Discord currently does for me.

**Describe alternatives you've considered**
It's certainly possible to click down the list of unread rooms, find the earliest unread message, and work that way. But it isn't as nice as Discord's inbox.

**Additional context**
I'm a blind screen reader user, so there are extra complications in reading room-by-room. In particular, I have to bump up the delay before the read marker disappears, and do bunches of keypresses to switch between panels. I recently had to decide between Matrix and Discord for a chat community I was creating, and the lack of an inbox-like feature meant I was less likely to keep on top of my own community. That was a deciding factor, and I hope some sort of inbox/timeline feature makes it into a future version of Element.

**Comment by ndarilek** [comment]
Right, but clearing or not is orthogonal. Let's sketch a scenario.

1. I join #kitties:pets.org
2. I join #puppies:pets.org.
3. I join #general:conduit.rs.

Right now, to keep up with all of those, I do:

1. Press ctrl-k.
2. Arrow down to `#puppies`.
3. Press enter.
4. Press s.
5. Read through messages.
6. Mark read.
7. Press ctrl-k.
8. Arrow down, probably multiple times, to #kitties:pets.org. Note that I also have to remember what I've read, which may get a bit dicey after 5-6 rooms when they're all reordering.
9. Press enter.
10. Press s.
11. Read, mark read.
12. ...

Compare that to joining multiple servers on Discord, with multiple 
active channels per room. It's a much larger stream of information, but 
the flow looks like:

1. Join the pets.org server. Note that, as per how Discord works, that automatically joins me to every channel on that server. That is orthogonal, I only mention it because by default Discord exposes me to a much larger river of information.
2. Join the conduit.rs Discord server.
3. Press ctrl-i. I'm automatically focused on the Inbox view, which contains all unread messages.
4. Read all unreads.
5. Click "Mark inbox as read." All unreads are now marked as read.
6. Optionally, while reading the inbox, press Ctrl-shift-e to mark messages above my current focus as read. This lets me come back later and pick up where I left off everywhere.

And again, because pets.org may have bunches of channels, this method 
makes it significantly easier to track a *much* larger volume of 
information. I actually have to trim things down on Discord to keep it 
manageable, but once I've got my core channels set, the UX for keeping 
up with 10-15 *entire Discord servers* is more pleasant to me than that 
of keeping up with a handful of Matrix rooms. Matrix has the big 
advantage of opting into rooms rather than opting out, so by default my 
inbox starts out manageable, and only becomes unmanageable when I've 
joined too many rooms.

I know this is probably low-priority, and that's fine. It just makes me 
a bit sad when every day I keep on top of this pile of Discord servers 
so much more easily, and when I try to do the same with Matrix in 
Element, it's so much more laborious. I want to be more active in 
Matrix, but right now the effort is higher than I have the bandwidth 
for. And while it may not be as common to treat Matrix like Discord in 
this regard, I wonder if that's inadvertently pushing some types of 
users toward other platforms that meet the particular need better. Right 
now, if some community has a Matrix or a Discord presence, I'd probably 
choose Matrix for this specific reason.

Thanks.

**Comment by ajkessel** [comment]
It would also be useful to have an optional virtual space in Element that collects all rooms with unread messages, in addition to "favorites," "people," and "other rooms". I think that is within the spirit of the request/suggestion here, but if you think it is a separate idea, I can file another issue.

**Comment by ndarilek** [comment]
I'd be fine with the virtual space, though that's only a part of what I'd like. The virtual space + some sort of view of all unread messages in that space is ultimately what I'm after.

**Comment by jivanpal** [comment]
> There is a Notifications pane in Element, aria-label `Notifications` which turns the right panel into a list of your notifications in all rooms.

This no longer exists, right? Element Desktop 1.11.46 seems to lack it, at least, but I do remember it being there, just not very useful in my experience.

**Comment by t3chguy** [comment]
@jivanpal [see releases notes](https://github.com/vector-im/element-web/releases/tag/v1.11.44).
